sys/kern/kern_malloc.c

```diff
@@ -1482,6 +1482,30 @@ SYSCTL_OID(_kern, OID_AUTO, mprof,
     sysctl_kern_mprof, "A",
     "Malloc Profiling");
 #endif /* MALLOC_PROFILE */
+
```

Comment:
Strictly speaking this is only a debugging hack, but @nwf may find it useful to use for getting vm_page_reclaim_run() to run in the future if he wants to exercise this path.  Probably this change should just hang out on some random branch somewhere and not in master/dev.